sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Can you provide a reference documentation link to the valid ML domains and tasks, in case support for new ones is added in the future?</p><p> </p><p>Are these "examples", or a list of values that are supported by Inference Recommender? I'm a little confused whether the customer can pass in any string describing their ML use case, or if these are enums.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
There is not a documentation exist for valid ML domains and tasks
The way I do it in the notebook is by `using client.list_model_metadata` which will fetch all the frameworks, tasks and model names supported.